### ISSUE TITLE
Property tags is not allowed. yaml-schema: Ansible Execution Environment Schema v3

### ISSUE THREAD

**Issue by alexander-aps** [ansible/ansible-lint#4343]
##### Summary

ansible-lint 24.9.2 reports incorrect information when including "tags" within "options".

This [is in the official documentation](https://ansible.readthedocs.io/projects/builder/en/latest/definition/#options) and works correctly when you build the EE.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint 24.9.2 using ansible-core:2.17.4 ansible-compat:24.9.1 ruamel-yaml:0.18.6 ruamel-yaml-clib:0.2.8
```

- OS: Red Hat Enterprise Linux release 9.4
- Python Version: 3.12
- PIP Version: 24.2
- ansible installation method: pip (venv)
- ansible-lint installation method: pip (venv)

##### STEPS TO REPRODUCE

In VSCode, with the ansible-lint path correctly updated, create an `execution-environment.yml` file and include the `tags` section within `options`

This will cause ansible-lint to flag `tags` as an error.

```yaml
options:
  package_manager_path: /usr/bin/microdnf
  tags:
    - automation-hub.my.company.com/ee-supported-rhel9_azure:latest
```

##### Desired Behavior

I expect this not to be flagged as an error.

##### Actual Behavior

It gets flagged as an error.
